infra/sql/atlas_security.sql

```diff
@@ -0,0 +1,48 @@
+create schema IF NOT EXISTS webapi_security;
+
+DROP TABLE IF EXISTS webapi_security.security;
+
+CREATE TABLE webapi_security.security
+(
+    email character varying(255),
+    password character varying(255)
+);
+
+insert into webapi_security.security (email,password) values ('admin', '$2a$10$opEKwT32fEvoPfSbzE1Rx.p8QsCG0KryiA7VEguLP/V0M62aho6mC');
```

Comment:
Is this hash means the password is hardcoded?

Reply:
yes, for now.